mimic_constraint/proposal.md

```diff
@@ -0,0 +1,318 @@
+# Proposal for Mimic Joint Actuation Constraint
+
+* **Authors**:
+Steve Peters`<scpeters@openrobotics.org>`,
+Aditya Pande `<aditya.pande@osrfoundation.org>`
+* **Status**: *Draft*
+* **SDFormat Version**: *1.10+*
```

Comment:
Should this be 1.11?

Reply:
we had been targeting garden / 1.10 since October 2021, but I can retarget to Harmonic / 1.11. I'll need to update the other pull requests; please confirm

Reply:
changed to 1.11 in https://github.com/gazebosim/sdf_tutorials/pull/62/commits/57aaca17bc43cffbe62f16ab62ba9655f8c52723; I will proceed with retargeting the other pull requests